test/testsets/GlassCat.jl

```diff
@@ -2,160 +2,128 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # See LICENSE in the project root for full license information.
 
+using Test
+using OpticSim
+
+using Unitful
+using Base.Filesystem
+using DataFrames
+using StaticArrays
```

Comment:
I find this setup quite useful as it allows me to just run `include("test/testsets/GlassCat.jl")`